configure

```diff
@@ -1063,7 +1063,30 @@ EOF
     fi
 }
 
-check_sse4_intrinsics() {
+check_sse41_intrinsics() {
+    # Check whether compiler supports SSSE3 intrinsics
+    cat > $test.c << EOF
+#include <x86intrin.h>
```

Comment:
`#include <immintrin.h>`?

Reply:
Actually I think with sse4 it's supposed to be the pmmintrin, xmmintrin, etc specific includes.  Though, immintrin.h might work?